### ISSUE TITLE
Tumblr Alt Text Support ?

### ISSUE THREAD

**Issue by rowantobias** [mvdicarlo/postybirb#324]
i saw an issue from a year ago theorizing it, though the addition of alt text as an option has not actually been added since tumblr created a bespoke alt text option. for some reason i can't add it myself if postybirb posted it because it uses the Old Post Editor(tm) and i can't update the post to the New Post Editor(tm) to fix it either.

the postybirb-posted images with the old layout (no alt text option, no option to update post type)
![old](https://github.com/user-attachments/assets/1988ebba-b5a7-4782-82df-1e1b42375d1a)

manually posted image in the new layout ("update image description" is the alt text option)
![new](https://github.com/user-attachments/assets/f4fd6d41-b6ab-410c-81d9-40e663b3ea34)

**Comment by leaftail1880** [comment]
There is no way to post images with alt text using tumblr api, even for the new post type

**Comment by rowantobias** [comment]
ack, for real?? that's so weird. tumblr made of spaghetti as always....

**Comment by leaftail1880** [comment]
ya tumblr is very limited and its sad actually

i guess using [Image ID ... End ID] is a way, as you already said on your blog

**Comment by rowantobias** [comment]
i wouldn't mind the lack of alt text if it were possible to at least convert it to new post editor format so i could add it myself minus add/end id in the post itself. would that potentially be possible, like, making it use new post editor instead of old via postybirb?
